extension/src/popup/views/Account/index.tsx

```diff
@@ -158,13 +134,12 @@ export const Account = () => {
             </div>
           </div>
         </div>
-        <div>
+        <div className="AccountView__assets-wrapper">
           {isFunded ? (
             <>
               <AccountAssets
                 sortedBalances={sortedBalances}
                 assetIcons={assetIcons}
```

Comment:
to clarify what I was saying up above: do we need to bother setting `assetIcons` here and passing to `AccountAssets`? Should we just fetch/retry `assetIcons` from within `AccountAssets`?